.gitignore

```diff
@@ -1,3 +1,5 @@
+training/training_output/
+*-model-building-code.zip
```

Comment:
would you mind adding `.idea/` to `.gitignore` while you're at it?

Reply:
Of course!